### PR TITLE
Sprint 2 - Import/Export CSV

### PR DESCRIPTION
**Pull request by Ay1tsMe**
Current functionality is to export all tables to csv's and zip them to a `database.zip`. Only tested functionality for the `student` table. Need to test other tables when we have code that populates them

TODO:
- Test other tables that aren't `student`
- Add button to frontend that gives user the database.zip file (waiting for `/admin` page)

### CODE REVIEW

**Review by DeltaO3 — CHANGES_REQUESTED** [review]
Looks great, I feel this and your other issue are similar enough that you could just merge them as one. Again, as my comment in other PR said:
Maybe move student_exists to database.py,
large edit of database file (merge conflict)? makes history hard to track

File: app/utilities.py [hunk 1]
```diff
@@ -24,33 +29,109 @@ def read_csv_file(file_path):
...
+        # Read the data from the CSV file
+        data = read_csv_file(file_path)
+        if data:
+            # Import the data to the database
+            import_student_in_db(data)
+
+# Export a single table's data to a CSV format and return it as a string
+def export_table_to_csv(model):
+    # Query all records from the model's table
+    records = db.session.query(model).all()
```
Comment (DeltaO3) [inline app/utilities.py:72]: also, pretty sure you can use the database function and pass no parameters to get the whole table?

Comment (Ay1tsMe) [reply app/utilities.py]: Now using `database.py` functions

**Review by DeltaO3 — APPROVED** [review]
LGTM!

**Pull request merged**